service/controller/v18/key/key.go

```diff
@@ -48,10 +48,10 @@ const (
 
 	FlannelEnvPathPrefix = "/run/flannel"
 	CoreosImageDir       = "/var/lib/coreos-kvm-images"
-	CoreosVersion        = "1855.5.0"
+	CoreosVersion        = "1911.5.0"
```

Comment:
@giantswarm/sig-operator why do we define that here? Don't we do that in `k8scloudconfig`?

Reply:
I have mixed feelings. E.g. in AWS we need to know the AMI. I wouldn't put that info into `k8scloudconfig`. Also releases arrive on different time on different providers and from time to time we need bugfixes specifically for a provider.